build.gradle

```diff
@@ -20,6 +20,8 @@ allprojects {
         flatDir {
             dirs '../libs'
         }
+        // Used in case when testing local Core SDK snapshot releases
+//        maven { url '/Users/aleksandr.chatsky/Projects/glia-android-sdk/build/maven_repo' }
```

Comment:
I think we shouldn't release it to the public repository because it's your local path and it won't work for others, isn't it?

Reply:
Fixed in cf59e63